### DISCUSSION TITLE
Any ideas for finetuning?

### DISCUSSION THREAD

**Discussion by d22b** [awawa-dev/HyperSerialESP32#51] [Q&A]
Hi awawa,

I followed your awesome 2023 guide and was able to setup an multisegment setup via HyperSPI. Everything works and look great. However, the different checkboxes in the software seems to be have some space for improvement.

Any idea what I should check on for the „best user experience“?
![HyperHDR - Webkonfiguration2](https://github.com/awawa-dev/HyperSerialESP32/assets/124438849/0922fa73-7549-46aa-a09d-4528a8dd049c)
![HyperHDR - Webkonfiguration](https://github.com/awawa-dev/HyperSerialESP32/assets/124438849/bf14817d-cb8d-49f9-9d6f-4bbd931499f1)
![HyperHDR - Webkonfiguration4](https://github.com/awawa-dev/HyperSerialESP32/assets/124438849/8d8121e5-c2fe-4c2a-ad87-e76960750a22)
![HyperHDR - Webkonfiguration3](https://github.com/awawa-dev/HyperSerialESP32/assets/124438849/8dcc5186-d3c4-495f-9a8c-7c36aad3caa5)

**Comment by d22b** [comment]
Well I am using the god old MS2109 and the SK6812 strip RGBWCW.

**Comment by awawa-dev** [comment]
Hi
I don't understand anything in this language at all, but if I remember the arrangement of controls correctly, it's probably OK ;) In my setup I do not use the blackbar detection so not sure about it. Maybe I wouldn't use such a high video resolution. Rather lower resolution with higher FPS. `quarter of frame` is not completely free in terms of resources used, a downscaler on the grabber's side is better if it has it by selecting a lower resolution. You can also calibrate the LUT using the selected codec (MJPEG or YUV) and the colorspace your videoplayer uses (e.g. YUV 444). The question is why do you want to continue tuning the current system, what is wrong?

BTW, I have recently been experimenting with the calibration of the white channel for HyperSPI: lowering its maximum brightness in its options in HyperHDR->HyperSPI LED driver gives more expressive colors, although you cannot overdo it.
